pkg/api/controllers/account_controller.go

```diff
@@ -78,7 +78,9 @@ func (ctl *AccountController) GetAccount(c *gin.Context) {
 func (ctl *AccountController) PostAccountMetadata(c *gin.Context) {
 	l, _ := c.Get("ledger")
 	var m core.Metadata
-	c.ShouldBind(&m)
```

Comment:
We should probably respond with a 400 if the "ShouldBind" function fail.

Reply:
Yes it should be better indeed.